src/tools/text/textconfig.cpp

```diff
@@ -152,25 +152,24 @@ void TextConfig::weightButtonPressed(const bool w)
 
 void TextConfig::setTextAlignment(Qt::AlignmentFlag alignment)
 {
-
     switch (alignment) {
         case (Qt::AlignLeft):
             m_leftAlignButton->setChecked(true);
             m_centerAlignButton->setChecked(false);
             m_rightAlignButton->setChecked(false);
             break;
-
         case (Qt::AlignCenter):
             m_leftAlignButton->setChecked(false);
             m_centerAlignButton->setChecked(true);
             m_rightAlignButton->setChecked(false);
             break;
-
         case (Qt::AlignRight):
             m_leftAlignButton->setChecked(false);
             m_centerAlignButton->setChecked(false);
             m_rightAlignButton->setChecked(true);
             break;
+        default:
```

Comment:
can you make the default case the same as Qt::AlignLeft? That will "solve" the warning clang-tidy was giving, this change will make the warning stop emitting but doesn't change the behavior.

Reply:
Done